spring-cloud-commons/src/main/java/org/springframework/cloud/client/loadbalancer/LoadBalancerProperties.java

```diff
@@ -136,6 +149,23 @@ public void setAddServiceInstanceCookie(boolean addServiceInstanceCookie) {
 
 	}
 
+	/**
+	 * To add X-forwarded headers.
+	 */
+	public static class Xforwarded {
+
+		private boolean addxforwarded = true;
```

Comment:
This is awkwardly named. The property would be `spring.cloud.loadbalancer.xforwarded.addxforwarded=true`. Maybe just `enabled`?